app/res/drawable/progressbar_modern_blue.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:id="@android:id/background">
+        <shape>
+            <corners android:radius="2dp" />
```

Comment:
@pm-dimagi looking great, can you just make sure that the radius of both the background and the bar match, otherwise there will be a gap:
![image](https://github.com/user-attachments/assets/6f04a242-665f-420b-a2d2-eab412b91ad0)